app/src/main/java/com/github/se/orator/ui/offline/OfflineInterviewModule.kt

```diff
@@ -56,78 +61,124 @@ fun OfflineInterviewModule(
+                          focusedBorderColor = MaterialTheme.colorScheme.outline,
+                          unfocusedBorderColor = MaterialTheme.colorScheme.outlineVariant,
+                          cursorColor = MaterialTheme.colorScheme.primary,
+                          focusedLabelColor = MaterialTheme.colorScheme.onSurface,
+                          unfocusedLabelColor = MaterialTheme.colorScheme.onSurfaceVariant))
 
-        Button(
-            onClick = {
-              // this is for the profile screen to see previous interviews
-              offlinePromptsFunctions.savePromptsToFile(
-                  context = context,
-                  prompts =
-                      mapOf(
-                          "targetCompany" to targetCompany,
-                          "jobPosition" to jobPosition,
-                          "ID" to ID,
-                          "transcribed" to "0",
-                          "GPTresponse" to "0",
-                          "transcription" to ""),
-              )
-              offlinePromptsFunctions.createEmptyPromptFile(context, ID)
-              speakingViewModel.interviewPromptNb.value = ID
-              navigationActions.navigateTo(Screen.PRACTICE_QUESTIONS_SCREEN)
-            },
-            modifier =
-                Modifier.fillMaxWidth(0.8f)
-                    .padding(AppDimensions.paddingSmall)
-                    .testTag("doneButton")
-                    .align(Alignment.CenterHorizontally),
-            colors = ButtonDefaults.buttonColors(containerColor = colors.primary)) {
-              Text(
-                  text = "Go to questions screen",
-                  fontSize = AppFontSizes.buttonText,
-                  color = colors.onPrimary)
+              OutlinedTextField(
+                  value = question,
+                  onValueChange = { newQuestion -> question = newQuestion },
+                  label = {
+                    Text("What do you want to focus on?", modifier = Modifier.testTag("question"))
+                  },
+                  modifier = Modifier.fillMaxWidth().testTag("question_field"),
+                  singleLine = false,
+                  colors =
+                      androidx.compose.material.TextFieldDefaults.outlinedTextFieldColors(
+                          backgroundColor = MaterialTheme.colorScheme.surface,
+                          textColor = MaterialTheme.colorScheme.onSurface,
+                          focusedBorderColor = MaterialTheme.colorScheme.outline,
+                          unfocusedBorderColor = MaterialTheme.colorScheme.outlineVariant,
+                          cursorColor = MaterialTheme.colorScheme.primary,
+                          focusedLabelColor = MaterialTheme.colorScheme.onSurface,
+                          unfocusedLabelColor = MaterialTheme.colorScheme.onSurfaceVariant))
+
+              Spacer(modifier = Modifier.height(AppDimensions.paddingLarge))
+
+              Button(
```

Comment:
Starting from the 2nd trial, this button keeps crashing the app.

Reply:
Indeed. Here is the error message : 
java.lang.IllegalArgumentException: Navigation destination that matches request NavDeepLinkRequest{ uri=android-app://androidx.navigation/offline_recording/ } cannot be found in the navigation graph ComposeNavGraph(0x0) startDestination={ComposeNavGraph(0x78c6a2d5) route=Auth startDestination={Destination(0x594b8837) route=Auth Screen}}

Reply:
You can try now, it should be fixed.